wheatley/bot.py

```diff
@@ -272,29 +275,29 @@ def tick(self) -> None:
 
             # Implement handbell-style 'up down in'
             if self._do_up_down_in and self._is_ringing_rounds and self._row_number == 2:
+                self.logger.debug("Setting '_should_start_method' for handbell-style start.")
                 self._should_start_method = True
 
             # Implement handbell-style stopping at rounds
-            if self._stop_at_rounds and has_just_rung_rounds:
+            if self._stop_at_rounds and has_just_rung_rounds and not self._is_ringing_rounds:
                 self._should_stand = False
                 self._is_ringing = False
 
             # If we're starting a handstroke, we should convert all the flags into actions
-            if self._row_number % 2 == 0:
-                if self._should_stand:
-                    self._should_stand = False
-                    self._is_ringing = False
+            if self._should_stand:
+                self._should_stand = False
+                self._is_ringing = False
+
+            if self._should_start_ringing_rounds and not self._is_ringing_rounds:
+                self._should_start_ringing_rounds = False
+                self._is_ringing_rounds = True
```

Comment:
I think these should still be just before handstroke for the moment? Otherwise won't Wheatley would normally stand with the bell at backstroke.
I wanted to add a time check to try and solve #49 once we've stopped churning through this file, but that should probably also be at handstroke.

Reply:
OK so I pushed these changes halfway through development (just before I went to bed).  I'm partway through refactoring this part of the code, so this will be fixed.